src/frontend/planner_test/tests/testdata/nexmark.yaml

```diff
@@ -678,7 +678,7 @@
     LogicalProject { exprs: [auction.id, auction.item_name, auction.description, auction.initial_bid, auction.reserve, auction.date_time, auction.expires, auction.seller, auction.category, bid.auction, bid.bidder, bid.price, bid.date_time] }
     └─LogicalFilter { predicate: (row_number <= 1:Int32) }
       └─LogicalProject { exprs: [auction.id, auction.item_name, auction.description, auction.initial_bid, auction.reserve, auction.date_time, auction.expires, auction.seller, auction.category, bid.auction, bid.bidder, bid.price, bid.date_time, row_number] }
-        └─LogicalOverAgg { window_functions: [row_number() OVER(PARTITION BY auction.id ORDER BY bid.price DESC, bid.date_time ASC)] }
+        └─LogicalOverAgg { window_functions: [row_number() OVER(PARTITION BY auction.id ORDER BY bid.price DESC, bid.date_time ASC ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING)] }
```

Comment:
A little bit difficult for now, because the default case is determined according to `ORDER BY`. Later when we support `Frame::Range`, we can use `Frame::Range(Unbounded, CurrentRow)` as default frame, to avoid this.